cli/command/container/run_test.go

```diff
@@ -202,6 +204,7 @@ func TestRunAttachTermination(t *testing.T) {
 	}
 
 	assert.NilError(t, syscall.Kill(syscall.Getpid(), syscall.SIGTERM))
+	conn.Close()
```

Comment:
are you seeing problems when conn isn't closed here? I ran multiple tests and saw no impact when removing it

Reply:
Yes, it worked because of https://github.com/docker/cli/commit/30c4637f03ecb8855fe6e47f5374c89d50222577, but the bug is fixed with https://github.com/docker/cli/pull/5957/commits/24e012b7777d1f9026a0f80b30a69269805e2f79 so the test fails.

Reply:
ah right, sorry didn't look in too much detail here. Makes sense 👍